tests/unit/utilities/valid-project-name-test.js

```diff
@@ -5,17 +5,15 @@ const expect = require('chai').expect;
 
 describe('validate project name', function() {
   it('invalidates nonconformant project name', function() {
-    let nonConformantName = 'app';
-    let validated = validProjectName(nonConformantName);
-
-    expect(validated).to.not.be.ok;
+    ['app', '.', 'ember', 'so-cool.', 'vendor', '1234test'].forEach(name => {
+      expect(
+        validProjectName(name),
+        `'${name}' is an invalid name`
+      ).to.not.be.ok;
+    });
```

Comment:
I usually prefer to generate complete tests instead of assertions, that way you can use `'${name}' is an invalid name` as the test name and don't have to add it to the assertions